riak_test/log_handoff_test.erl

```diff
@@ -40,6 +43,11 @@ confirm() ->
     [RootNode | TestNodes] = rt:deploy_nodes(Versions,[logging]),
     rt:wait_for_service(RootNode, logging),
 
+
+    lager:info("Waiting until vnodes are started up"),
```

Comment:
If the vnodes are not started then the test will crash.  This is because the reads happen from the shared ets table that is owned by the vnode, but can be accessed by other concurrent reader threads.

An alternative could be to add the check within the read operation instead of during startup.  This would probably be better for safety.

I guess I have it the way it is currently because as far as I understand consistency isn't insured during handoff, so I am assuming when we look at how we will ensure consistency during handoff this issue will be solved as well since startup is sort of a sub issue of handoff.